<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>781</width>
     <height>701</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Configuracion</string>
    </attribute>
    <widget class="QLabel" name="label_27">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Buscar dispositivos Bluetooth</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inicio_7">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>101</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(213, 213, 213);
background-color: rgb(11, 0, 172);
</string>
     </property>
     <property name="text">
      <string>Buscar BLE</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selec_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="selec2">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>20</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>101</width>
       <height>41</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(168, 168, 168);</string>
     </property>
     <property name="text">
      <string>Selecionar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>120</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Parametros </string>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_acc_sampling">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Acc Sampling</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Acc Sensibility</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_acc_sensibity">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>200</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="gxmax_5">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>250</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_39">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>260</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Gyro Sensibility</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_41">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>160</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>BME688 Sampling</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="gxmax_8">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>150</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_42">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>210</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Discontinuos time</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="gxmax_9">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>200</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="gxmax_13">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>250</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_54">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Port TCP</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_55">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>160</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Port UDP</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="gxmax_14">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>150</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="gxmax_22">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>250</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_70">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>260</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Host_ip_addr</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="gxmax_31">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>150</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_86">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>160</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Ssid</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_87">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>210</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Pass</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="gxmax_32">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>200</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inicio">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>410</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>101</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(213, 213, 213);
background-color: rgb(0, 115, 0);
</string>
     </property>
     <property name="text">
      <string>Save Conf</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_28">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>310</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Protocolo</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selec_8">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>340</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="server_start_button">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>530</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>101</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(213, 213, 213);
background-color: rgb(0, 115, 0);
</string>
     </property>
     <property name="text">
      <string>Iniciar Server</string>
     </property>
    </widget>
    <widget class="QPushButton" name="server_stop">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>530</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>101</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(213, 213, 213);
background-color: rgb(108, 5, 5);
</string>
     </property>
     <property name="text">
      <string>Detener Monitoreo</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selec_6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>340</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>Configuracion por Bluethooth</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Configuracion via TCP en BD</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Conexion TCP continua</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Conexion TCP discontinua</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Conexion UDP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BLE continua</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BLE discontinua</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_26">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>310</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Modo de operacion (status)</string>
     </property>
    </widget>
    <widget class="QComboBox" name="servers_active_stop">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>540</y>
       <width>201</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>520</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Port</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>520</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Protocol</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="server_port">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>540</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100000</number>
     </property>
     <property name="value">
      <number>5000</number>
     </property>
    </widget>
    <widget class="QComboBox" name="server_status">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>540</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>20</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>21</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>22</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>23</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>31</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="servers_active_update">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>590</y>
       <width>211</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="server_update_button">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>580</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>101</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(213, 213, 213);
background-color: rgb(0, 115, 0);
</string>
     </property>
     <property name="text">
      <string>Updatear Server</string>
     </property>
    </widget>
    <widget class="QComboBox" name="server_protocol">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>540</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>0</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>520</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Visualizacion</string>
    </attribute>
    <widget class="GraphicsLayoutWidget" name="plot1">
     <property name="geometry">
      <rect>
       <x>-10</x>
       <y>30</y>
       <width>501</width>
       <height>461</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inicio_5">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>90</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>101</width>
       <height>41</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(213, 213, 213);
background-color: rgb(0, 115, 0);
</string>
     </property>
     <property name="text">
      <string>Graficar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>60</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>Variable</string>
     </property>
    </widget>
    <widget class="QComboBox" name="selec">
     <property name="geometry">
      <rect>
       <x>560</x>
       <y>60</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <item>
      <property name="text">
       <string>Temperatura</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Humedad</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Co</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Presion</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RMS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Bateria</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Frecuencia X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Frecuencia Y</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Frecuencia Z</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Amplitud X</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Amplitud Y</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Amplitud Z</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphicsLayoutWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
